micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/metrics/config/IsGraphitePublishingEnabled.groovy

```diff
@@ -0,0 +1,22 @@
+package com.ofg.infrastructure.metrics.config
+
+import org.springframework.context.annotation.Condition
+import org.springframework.context.annotation.ConditionContext
+import org.springframework.core.type.AnnotatedTypeMetadata
+
+class IsGraphitePublishingEnabled implements Condition {
+
+    @Override
+    boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata) {
+        return graphitePublishingSwitchNotDefined(context) || graphitePublishingIsEnabled(context)
+    }
+
+    private boolean graphitePublishingSwitchNotDefined(ConditionContext context) {
+        !context.environment.containsProperty('graphite-publishing')
```

Comment:
Exactly. You need to provide this flag manually if you'd like to switch off publishing (and avoid exceptions in log).